Helper/Url.php

```diff
@@ -49,6 +49,7 @@
 use Nosto\Tagging\Helper\Data as NostoDataHelper;
 use Nosto\Tagging\Model\Product\Repository as ProductRepository;
 use Nosto\Tagging\Model\Product\Url\Builder as NostoUrlBuilder;
+use Nosto\Helper\UrlHelper;
```

Comment:
This class doesn't exist? Or am I missing something?

Reply:
SDK version 3.10.1